smoke-tests/apps/DetectUnexpectedOtelMetrics/src/smokeTest/resources/applicationinsights.json

```diff
@@ -0,0 +1,31 @@
+{
+  "role": {
+    "name": "testrolename",
+    "instance": "testroleinstance"
+  },
+  "sampling": {
+    "percentage": 100
+  },
+  "preview": {
+    "roleNameOverrides": [
+      {
+        "httpPathPrefix": "/DetectUnexpectedOtelMetrics/app2",
+        "roleName": "app2"
+      },
+      {
+        "httpPathPrefix": "/DetectUnexpectedOtelMetrics/app3",
+        "roleName": "app3"
+      }
+    ],
+    "connectionStringOverrides": [
+      {
+        "httpPathPrefix": "/DetectUnexpectedOtelMetrics/app2",
+        "connectionString": "InstrumentationKey=12345678-0000-0000-0000-0FEEDDADBEEF;IngestionEndpoint=http://host.testcontainers.internal:6060/"
+      },
+      {
+        "httpPathPrefix": "/DetectUnexpectedOtelMetrics/app3",
+        "connectionString": "InstrumentationKey=87654321-0000-0000-0000-0FEEDDADBEEF;IngestionEndpoint=http://host.testcontainers.internal:6060/"
+      }
+    ]
```

Comment:
does this test need the extra complexity of role overrides?

Reply:
initially, i was thinking of checking there is any rolename or ikey not being overwritten.